src/main/java/command/Find.java

```diff
@@ -0,0 +1,38 @@
+package command;
+
+import tasks.TaskList;
+import tasks.Task;
+
+import java.sql.Array;
+import java.util.Arrays;
+import java.util.List;
+
+public class Find extends Command{
```

Comment:
Maybe you can make the `Find` class sound more like a noun like this?
```suggestion
public class FindCommand extends Command {
```